### ISSUE TITLE
Add option to not normalize CIFAR10 images

### ISSUE THREAD

**Issue by rickwierenga** [tensorflow/swift-models#285]
I noticed that https://github.com/tensorflow/swift-models/blob/master/Datasets/CIFAR10/CIFAR10.swift always normalizes images. I would like to contribute a flag where the user can set `normalizing: false`.

The new function declaration would be:

```swift
loadCIFARFile(named name: String, in directory: URL, normalizing: Bool = false)
```

We should use a default value for `normalizing` so it is compatible with previous versions.

**Comment by sainikhit2k** [comment]
Hi ,  is this issue currently open ? If it is I would like to contribute to this issue

**Comment by rickwierenga** [comment]
@sainikhit2k, I think @manideep2510 is working on this in https://github.com/tensorflow/swift-models/pull/349

**Comment by BradLarson** [comment]
Fixed by PR #349.
